Dockerfile

```diff
@@ -17,15 +17,26 @@ RUN apk --no-cache add \
     texmf-dist-latexextra \
     texmf-dist-pictures \
     texmf-dist-science \
-    wget
+    wget \
+    curl \
+    git
 
 
 RUN luaotfload-tool --update
 RUN apk --no-cache add py3-pandas
-RUN apk --no-cache add pipx
-RUN pipx install cgt-calc
-RUN pipx ensurepath
 
-WORKDIR /data
+WORKDIR /build
+
+RUN curl -sSL https://install.python-poetry.org | python3 -
+RUN ln -s /root/.local/share/pypoetry/venv/bin/poetry /bin/
```

Comment:
I'm going to replace poetry and pipx with [uv](https://github.com/astral-sh/uv) soon.

Reply:
What difference does it make? I can't stop wondering to the wide amount of available and popular automatic packing tools in python that keep pulling some code from somewhere, compile and run it on my laptop!

Don't forget to update this Dockerfile thereafter then